v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToBigQueryFlex.java

```diff
@@ -156,24 +152,31 @@ public static void main(String[] args) throws IOException, RestClientException {
   public static PipelineResult run(KafkaToBigQueryFlexOptions options)
       throws IOException, RestClientException {
 
+    // Enable Streaming Engine
+    options.setEnableStreamingEngine(true);
+
+    // Enable Resource based billing
+    options.setDataflowServiceOptions(List.of("enable_streaming_engine_resource_based_billing"));
+
     // Validate BQ STORAGE_WRITE_API options
+    options.setUseStorageWriteApi(true);
+    if (options.getStorageWriteApiTriggeringFrequencySec() == null) {
+      options.setStorageWriteApiTriggeringFrequencySec(5);
+    }
     BigQueryIOUtils.validateBQStorageApiOptionsStreaming(options);
     MetadataValidator.validate(options);
 
     // Create the pipeline
     Pipeline pipeline = Pipeline.create(options);
 
     List<String> topicsList;
-    if (options.getKafkaReadTopics() != null) {
-      topicsList = new ArrayList<>(Arrays.asList(options.getKafkaReadTopics().split(",")));
-    } else {
-      throw new IllegalArgumentException("Please Provide --kafkaReadTopic");
-    }
     String bootstrapServers;
-    if (options.getReadBootstrapServers() != null) {
-      bootstrapServers = options.getReadBootstrapServers();
+    if (options.getReadBootstrapServerAndTopic() != null) {
+      String[] l = options.getReadBootstrapServerAndTopic().split(":");
+      topicsList = List.of(l[2]);
+      bootstrapServers = l[0] + ":" + l[1];
     } else {
-      throw new IllegalArgumentException("Please Provide --bootstrapServers");
+      throw new IllegalArgumentException("Please provide bootstrap server and topic.");
```

Comment:
I'd make the the error message more specific. Mention that the parameter value doesn't match a specific pattern (otherwise you may set it to something in an invalid format and still get "please provide the param" error, which is confusing).

Reply:
done